src/styles/stopwatch__minute-hand.scss

```diff
@@ -0,0 +1,21 @@
+@import 'variables';
+
+.stopwatch__minute-hand {
+  position: absolute;
+  width: $minute-hand-width;
+  height: $minute-hand-length;
+  background-color: $minute-hand-color;
+  top: calc(50% - #{$minute-hand-length});
+  left: 48,5%;
```

Comment:
There is a syntax error in the `left` property value. The comma should be replaced with a period to correctly represent a decimal value. Change `48,5%` to `48.5%`.